test/e2e/websocket_test.go

```diff
@@ -39,6 +39,7 @@ import (
 
 const (
 	wsServerTestImageName = "wsserver"
+	TimeoutNotSet         = ""
```

Comment:
If we use `sleep` or `delay` above as commented above, we should change this value to `sleep  = 0` or `delay = 0`.

Reply:
updated